compile.js

```diff
@@ -64,5 +68,8 @@ recursive('./src/model', function (err, files) {
         fs.copy('./src/styles', './dist/styles');
         fs.copy('./src/index.html', './dist/index.html');
         fs.copy('./src/favicon.ico', './dist/favicon.ico');
+
+        fs.copy('./src/__init__.py_base', './py_api_raml/__init__.py'); // for py
+        fs.copy('./dist/model', './py_api_raml'); // for py
```

Comment:
to be aware of: once something is committed to `develop`, https://ci--alfred.elifesciences.org/job/library-api-raml/ builds it and commits the changes to `dist` on top of it. That's the commit that should be used by downstream projects.
Perhaps we need to add more entries to https://github.com/elifesciences/api-raml/blob/develop/Jenkinsfile#L19?

Reply:
Definitely needs to be added then: `def isThereANewCommit = elifeGitAutoCommit "Regenerated dist/ and dependencies", "dist/", "py_api_raml/`

Reply:
Agreed. I will change the name to `elife_api` as suggested by @thewilkybarkid and then add the  change to the Jenkinsfile.

Reply:
Now that I am not storing the JSON copies in the `elife_api` directory I don't think this is required.

Reply:
Correct, as the code added here won't be affected by the regeneration

Reply:
Regarding https://github.com/elifesciences/bot-lax-adaptor/blob/develop/install.sh#L17-L19 I think something along these lines would be needed by projects dependant on it in order to keep `elife_api` up to date. If the version value in the `setup.py` does not change then pip will skip it by default.

The alternative is to use the `--upgrade --force-reinstall` args when running pip, this will reinstall even if the version has not changed. The downside being that if we did this in line with `pip install -r requirements.txt` everything will get reinstalled.

Reply:
Correct to keep the `elife_api` version pinned, every build starting from the same project commit should get the same result.
The pattern we use for dependency updates is to automate opening a pull request upon certain events - every week or when the upstream gets a new commit if it's something that must be kept up to date.

See:
- https://github.com/elifesciences/bot-lax-adaptor/pulls?utf8=%E2%9C%93&q=is%3Apr%20api-raml%20 (sample api-raml proposed updates in bot-lax-adaptor)
- https://github.com/elifesciences/bot-lax-adaptor/blob/develop/Jenkinsfile.update-api-raml (different builds that is triggered to do the update)
- https://github.com/elifesciences/bot-lax-adaptor/blob/develop/update-api-raml.sh weird script that does what pip should be doing. We would run `--upgrade` in a separate build which will open a PR rather than run tests.